api/src/main/java/javax/enterprise/inject/spi/Prioritized.java

```diff
@@ -0,0 +1,21 @@
+package javax.enterprise.inject.spi;
+
+/**
+ * <p>
+ * Provides access to the priority value of prioritized elements.
```

Comment:
I would add some more information e.g. a reference to the priority ranges. Other than that this looks good to me.